models.rb

```diff
@@ -91,6 +107,10 @@ def spread(n)
     scores.find_by(rank: n).spread unless !scores.exists?(rank: n)
   end
 
+  def tie
```

Comment:
I'd prefer the name `tie_count` or similar, since this doesn't return 'a tie' but instead the number of ties for the highscore.

Reply:
True, fixed.